Kernel/ACPI/ACPIParser.cpp

```diff
@@ -58,10 +58,10 @@ ACPIParser::ACPIParser(bool usable)
     s_acpi_parser = this;
 }
 
-ACPI_RAW::SDTHeader* ACPIParser::find_table(const char*)
+PhysicalAddress ACPIParser::find_table(const char*)
 {
     kprintf("ACPI: Requested to search for a table, Abort!\n");
-    return nullptr;
+    return PhysicalAddress((uintptr_t) nullptr);
```

Comment:
It's pretty janky to pass `nullptr` to the constructor of `PhysicalAddress`. As you mentioned in IRC, change this to `return {};`

https://stackoverflow.com/questions/39487065/what-does-return-statement-mean-in-c11

Reply:
Sure will do, thanks :)